### DISCUSSION TITLE
Data provider to filter by either Status A or Status B not working

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6231] [Help] [answered]
Imagine I have a "Project" table with 4 possible status:
Open, Completed, Cancelled, Closed. 

Now I need a table that shows either "Cancelled" or "Closed" projects. I do this in the filtering:
![Screenshot 2022-06-08 at 3 41 02 PM](https://user-images.githubusercontent.com/1195573/172560194-2c84256a-0374-4645-90c2-ba9374a7240a.jpg)

Basically saying the status must not equal "Open" AND must not equal "Completed". 

Expected results is filtered results that show only "Cancelled" or "Closed".

Actual results is that the filtered results incorrectly show projects with status "Open" "Cancelled" "Closed".

Is this a bug?

A demo is here:
[Test double status-export-1654674181824.txt](https://github.com/Budibase/budibase/files/8859440/Test.double.status-export-1654674181824.txt)

**Comment by magickaito** [comment, marked as answer]
I found a duplicated issue here https://github.com/Budibase/budibase/issues/2585

**Reply by melohagan** [reply]
Thanks for finding and linking the issue!
